test/github-release.ts

```diff
@@ -85,68 +89,30 @@ describe('GitHubRelease', () => {
         .stub(release.gh, 'removeLabels')
         .withArgs(['autorelease: pending'], 1)
         .resolves();
+      return release;
+    };
 
-      const created = await release.run();
+    it('creates and labels release on GitHub', async () => {
+      const releaser = gitHubReleaserForVersion('1.0.3');
+      const created = await releaser.run();
       strictEqual(created!.tag_name, 'v1.0.3');
       strictEqual(created!.major, 1);
       strictEqual(created!.minor, 0);
       strictEqual(created!.patch, 3);
       strictEqual(created!.draft, false);
     });
 
+    it('creates and labels release on GitHub with invalid semver', async () => {
+      const releaser = gitHubReleaserForVersion('1A.B.C');
+      const created = await releaser.run();
+      expect(created).to.be.undefined;
+    });
+
```

Comment:
I was gaming the coverage a little for this PR by adding this (and realizing the mock setup should be DRY'd out). Kind of broke my own "single purpose" PR rule. Kept it as a separate commit so I can easily remove this change if it's too distracting from the main idea of a `ReleaseType`